app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt

```diff
@@ -15,4 +44,116 @@ fun TaskScreen(
     task: Task
 ) {
 
+}
+
+@Composable
+fun SportTaskForm() {
+    var supportsDistanceMetrics by remember { mutableStateOf(false) }
+    var title by remember { mutableStateOf("") }
+    var duration by remember { mutableStateOf("") }
+    var startDate by remember { mutableStateOf(Date()) }
+
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = Color.White)
+    ) {
+        Column(
+            modifier = Modifier
+                .padding(16.dp)
+                .fillMaxSize(),
+            verticalArrangement = Arrangement.Center,
+            //horizontalAlignment = Alignment.CenterHorizontally
+        ){
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.padding(bottom = 16.dp)
+            ) {
+            Checkbox(
+                checked = supportsDistanceMetrics,
+                onCheckedChange = { supportsDistanceMetrics = it },
+                modifier = Modifier.padding(bottom = 16.dp)
+            )
+            Text (
+                text = "Supports distance metrics",
+                modifier = Modifier.padding(start = 16.dp)
+            )}
```

Comment:
Line :77 has redundant space before opening brackets.
Separate components should be separated by a single empty line to increase readability.
Wrong padding, both `Checkbox` and `Text` components are within `Row` component's scope, you should increase their padding by 4 spaces (single TAB).